tests/kafkatest/services/kafka/kafka.py

```diff
@@ -784,17 +784,12 @@ def prop_file(self, node):
         for prop in self.per_node_server_prop_overrides.get(self.idx(node), []):
             override_configs[prop[0]] = prop[1]
 
-        enabledProtocols = 'classic,consumer'
-
         if self.use_share_groups is not None and self.use_share_groups is True:
             override_configs[config_property.SHARE_GROUP_ENABLE] = str(self.use_share_groups)
-            enabledProtocols += ',share'
 
         if self.use_streams_groups is True:
             override_configs[config_property.UNSTABLE_API_VERSIONS_ENABLE] = str(True)
-            enabledProtocols += ',streams'
-
-        override_configs[config_property.GROUP_COORDINATOR_REBALANCE_PROTOCOLS] = enabledProtocols
```

Comment:
So we don't use `kafka.py` anymore for KIP-1071 related tests?

Reply:
We still are. We are just using the new feature flag that @mjsax introduced here: https://github.com/apache/kafka/pull/19604/files#diff-6a398e21e2938aac5ecfe4405096ed671320550fdb9d0563efc66ea008752908R50